team-telnyx/telnyx-node
# DOTCOM-3583: convert base files to typescript

- [ ] resources (wip, `Balance`)
- [x] Error
- [x] autoPagination
- [x] makeRequest
- [x] multipart
- [x] telnyx
- [x] TelnyxMethod
- [x] TelnyxResource
- [x] Webhooks

## Manual testing

- run `npm run lint`
- run `npm run build`
- check types in `types` folder
- check output in `dist` folder
- check README instructions on typescript

## Notes

keeping changes from previous #177 here to keep consistency as `v2.0.0` branch will be serving the published NODE SDK alpha versions before it's approved by users and merged to main
